### PR TITLE
Feature/protocol update

### PR DESCRIPTION
**Pull request by daneduijnkerke**

### CODE REVIEW

File: ci/indy-pool_old.dockerfile [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# This dockerfile originates from indy-sdk: https://github.com/hyperledger/indy-sdk/blob/8d4917fbe1c3e49c8bb42b21e15915c5f5459c30/ci/indy-pool.dockerfile
```
Comment (pimotte) [inline ci/indy-pool_old.dockerfile]: Just delete this, version control can be used to get it back.

File: ci/indy-pool.dockerfile [hunk 2]
```diff
@@ -1,5 +1,3 @@
-# This dockerfile originates from indy-sdk: https://github.com/hyperledger/indy-sdk/blob/8d4917fbe1c3e49c8bb42b21e15915c5f5459c30/ci/indy-pool.dockerfile
```
Comment (pimotte) [inline ci/indy-pool.dockerfile:1]: Please update the link as it was to point to the new revision.

File: src/main/java/nl/quintor/studybits/indy/wrapper/Issuer.java [hunk 3]
```diff
@@ -46,7 +46,9 @@ public Issuer(IndyWallet wallet) {
         return getSchema(getMainDid(), schemaId)
                 .thenCompose(wrapException(schema ->
                 {
-                    return Anoncreds.issuerCreateAndStoreCredentialDef(getWallet(), getMainDid(), schema.toJSON(), name + "_" + schemaId, "CL", "{\"support_revocation\":false}");
+                    // Remove special characters from the credential defenition tag
```
Comment (pimotte) [inline src/main/java/nl/quintor/studybits/indy/wrapper/Issuer.java]: Spelling defenition->definition

**Review by pimotte — CHANGES_REQUESTED** [review]

File: ci/indy-pool_old.dockerfile (hunk 1 above)
Comment (daneduijnkerke) [reply ci/indy-pool_old.dockerfile]: This line was part of the original dockerfile from the indy-sdk repository, are you sure this line should be deleted?

**Review by pimotte — APPROVED** [review]

**Pull request merged**